README.md

```diff
@@ -3,3 +3,60 @@
+| View public scenarios                                                                       |   ✅   |   ✅   |   ✅   |   ✅   |
+| Modify unlocked global inputs in unlocked <br> global scenarios without app storing values  |   ✅   |   ✅   |   ✅   |   ✅   |
+| About modal                                                                                 |   ❌   |   ✅   |   ✅   |   ✅   |
+| Simulations dropdown                                                                        |   ❌   |   ✅   |   ✅   |   ✅   |
+| Download sims                                                                               |   ❌   |   ✅   |   ✅   |   ✅   |
+| Scenarios dropdown                                                                          |   ❌   |   ✅   |   ✅   |   ✅   |
+| Bookmarks panel                                                                             |   ❌   |   ❌   |   ✅   |   ✅   |
+| Create scenarios                                                                            |   ❌   |   ❌   |   ✅   |   ✅   |
+| Edit scenarios                                                                              |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete scenarios                                                                            |   ❌   |   ❌   |   ✅   |   ✅   |
+| Duplicate scenarios                                                                         |   ❌   |   ❌   |   ✅   |   ✅   |
+| Run sims                                                                                    |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete sims                                                                                 |   ❌   |   ❌   |   ✅   |   ✅   |
+| Edit themes                                                                                 |   ❌   |   ❌   |   ✅   |   ✅   |
+| Copy/Delete Attributes                                                                      |   ❌   |   ❌   |   ✅   |   ✅   |
+| Delete owned bookmarks                                                                      |   ❌   |   ❌   |   ✅   |   ✅   |
+| Remove schema edits                                                                         |   ❌   |   ❌   |   ❌   |   ✅   |
+| Emulate permissions                                                                         |   ❌   |   ❌   |   ❌   |   ✅   |
+| Export non-Simulation layers                                                                |   ❌   |   ❌   |   ❌   |   ✅   |
+| Configurer modal                                                                            |   ❌   |   ❌   |   ❌   |   ✅   |
+| Download all scenarios                                                                      |   ❌   |   ❌   |   ❌   |   ✅   |
+| Show archived filters                                                                       |   ❌   |   ❌   |   ❌   |   ✅   |
+| Archive/unarchive filters                                                                   |   ❌   |   ❌   |   ❌   |   ✅   |
+
+
+### API
```

Comment:
@iancarlton do you want to document the api permissions?  I think I'd prefer not to.  Several of these endpoints are internal and undocumented and definitely shouldn't be here.